glutton.go

```diff
@@ -2,26 +2,24 @@ package glutton
 
 import (
 	"io/ioutil"
+	"net"
 	"os"
 	"path/filepath"
+	"strings"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/kung-foo/freki"
+	"github.com/mushorg/glutton/out"
 	uuid "github.com/satori/go.uuid"
 )
 
+var logHTTP *string
```

Comment:
If Boolean then I need to store remote address explicitly in a separate variable. I think there should be a type in logger package, instead of passing logger and address as argument to function I will pass to method. Let me try this :)